lib/chili_custom_lints.dart

```diff
@@ -1,12 +1,20 @@
 // This is the entrypoint of our custom linter
+import 'package:chili_custom_lints/lints/omit_explicit_parameters_types/omit_explicit_parameters_types_rule.dart';
 import 'package:chili_custom_lints/lints/rename_unused_params_to_underscore/rename_unused_params_to_underscore_rule.dart';
+import 'package:chili_custom_lints/lints/use_common_design_system_widget/use_common_design_system_empty_padding_lint_rule.dart';
+import 'package:chili_custom_lints/lints/use_common_design_system_widget/use_common_design_system_margin_lint_rule.dart';
+import 'package:chili_custom_lints/lints/use_common_design_system_widget/use_common_design_system_padding_lint_rule.dart';
 import 'package:custom_lint_builder/custom_lint_builder.dart';
 
 PluginBase createPlugin() => _ChiliCustomLinter();
 
 class _ChiliCustomLinter extends PluginBase {
   @override
   List<LintRule> getLintRules(CustomLintConfigs configs) => [
+        OmitFunctionParamsTypesRule(),
```

Comment:
I think we will need to disable it since there is already a rule for this use case. We need to enabled that one in the `chili_flutter_analysis`

Reply:
Keeping it as a learning reference for now. Added `Deprecated` annotation to the rule class so it's not accidentally added in future PRs. 

<img width="525" alt="Screenshot 2023-07-27 at 07 15 06" src="https://github.com/ChiliLabs/chili_custom_lints/assets/57680536/98ade172-1d70-4d06-844e-6e70083a9cdf">